warehouse/query-core/src/test/java/datawave/query/testframework/QueryParserHelper.java

```diff
@@ -0,0 +1,516 @@
+package datawave.query.testframework;
+
+import org.apache.log4j.Logger;
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.io.IOException;
+import java.lang.reflect.Type;
+import java.net.URI;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.Date;
+import java.util.HashSet;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Set;
+
+/**
+ * Helper class for parsing a query. Currently, the parser will only work properly on JEXL syntax. Queries that use functions or Lucene must be converted to an
+ * equivalent. An invalid query format will be rejected.
+ */
+public class QueryParserHelper {
```

Comment:
Why aren't we simply using a JexlParser from the jexl library?

Reply:
Not sure why I decided to do it this way. Perhaps I thought it would be simple enough. What I have works but I did encounter a few minor issues. Probably better to use the Apache Commons JEXL.

Reply:
The QueryParserHelper class has been removed and replaced by the QueryJexl class, which uses the Apache Jexl artifact.